Tests/Controllers/ParkingSlotControllerTests.cs

```diff
@@ -108,7 +110,93 @@ public void GivenNullParkingZoneId_WhenIndexIsCalled_ThenParkingZoneServiceIsCal
 
             //Assert
             Assert.IsType<BadRequestResult>(result);
-            mockParkingZoneService.Verify(s => s.GetById(Guid.Parse("")), Times.Once);
+            mockZoneService.Verify(s => s.GetById(Guid.Parse("")), Times.Once);
+        }
+        #endregion
+
+        #region Create
+        [Fact]
+        public void GivenParkingZoneId_WhenGetCreateIsCalled_ThenParkingZoneServiceIsCalledAndReturnedNotEmptyViewResult()
+        {
+            //Arrange
+            mockZoneService.Setup(service => service.GetById(_testZoneId)).Returns(_testZone);
+
+            //Act
+            var result = controller.Create(_testZoneId);
+
+            //Assert
+            Assert.IsType<ViewResult>(result);
+            Assert.NotNull((result as ViewResult).Model);
+            mockZoneService.Verify(s => s.GetById(_testZoneId), Times.Once);
+        }
```

Comment:
Add such assertion: `Assert.Equal(JsonSerializer.Serialize(expectedCreateSlotVM), JsonSerializer.Serialize((result as ViewResult).Model));`. See Dilshod's [PR](https://github.com/DilshodMQ/Parking-zone/pull/9/files#diff-0be201021cb04ff536474e37b39372b678b1d2ab895950064dab90d36337ae85R142) for details